### PR TITLE
Java Agent Attach- replacing "Lock" with Mutex for allowing sync also…

### PR DESCRIPTION
**Pull request by MeniKadosh1**
… between diffrent Ginger processes

### CODE REVIEW

File: Ginger/GingerCore/Actions/ActLaunchJavaWSApplication.cs [hunk 1]
```diff
@@ -325,10 +325,28 @@ public override void Execute()
...
+                            Reporter.ToLog(eLogLevel.WARN, "Attach Java Agent- Mutex Wait Timeout Reached");
+                        }
+                        else
+                        {
+                            Reporter.ToLog(eLogLevel.DEBUG, "Attach Java Agent- Mutex was Released");
+                        }
+                    }
+                    catch (Exception ex)
+                    {
+                        Reporter.ToLog(eLogLevel.WARN, "Attach Java Agent- Mutex Wait Threw Exception", ex); ;
```
Comment (ambarkhan) [inline Ginger/GingerCore/Actions/ActLaunchJavaWSApplication.cs]: Codacy Issue found: [Remove this empty statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5145905)

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5145905&bid=16902068)

**Comment by JinendraGhodke** [comment]
create new PR with all changes

**Comment by ambarkhan** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5145905&bid=16902068)

File: Ginger/GingerCore/Actions/ActLaunchJavaWSApplication.cs (hunk 1 above)
Comment (ambarkhan) [inline Ginger/GingerCore/Actions/ActLaunchJavaWSApplication.cs]: Codacy Issue found: [Remove this empty statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5145905)

**Pull request closed without merging**